core/contracts/Voting.sol

```diff
@@ -312,7 +308,7 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
     function getPrice(bytes32 identifier, uint time) external view onlyRegisteredDerivative() returns (int) {
         (bool _hasPrice, int price, string memory message) = _getPriceOrError(identifier, time);
 
-        // TODO(#779): If the price wasn't available, revert with the provided message.
+        //If the price wasn't available, revert with the provided message.
```

Comment:
Formatting nit: I don't think the linter cares, but we generally put a space between `//` and the comment.